qiskit_experiments/curve_analysis/curve_analysis.py

```diff
@@ -291,8 +309,11 @@ def _default_options(cls) -> Options:
             xlabel (str): X label of fit result figure.
             ylabel (str): Y label of fit result figure.
             ylim (Tuple[float, float]): Min and max height limit of fit plot.
-            fit_reports (Dict[str, str]): Mapping of fit parameters and representation
-                in the fit report.
+            db_parameters (Dict[str, Tuple[str, str]]): Parameters reported in the
```

Comment:
Maybe this should be `result_parameters` since it doesn't require DB. Also if you want to just use the parameter as its current name we probably only need to pass in the string, and a renaming and units is optional.